databuilder/databuilder/publisher/elasticsearch_publisher.py

```diff
@@ -124,5 +124,13 @@ def publish_impl(self) -> None:
         # perform alias update and index delete in single atomic operation
         self.elasticsearch_client.indices.update_aliases(update_action)
 
+    def close(self) -> None:
```

Comment:
why do we need this function? is it used anywhere?

Reply:
It's part of the parent class [Scoped](https://github.com/amundsen-io/amundsen/blob/3cc8bd587dd296d99ed94a5ee08a420be359249d/databuilder/databuilder/__init__.py#L56) and automatically executed at the end. We do the same already for [FSElasticsearchJSONLoader](https://github.com/amundsen-io/amundsen/blob/3cc8bd587dd296d99ed94a5ee08a420be359249d/databuilder/databuilder/loader/file_system_elasticsearch_json_loader.py#L59).